### PR TITLE
[Issue-23171] SSL_add_dir_cert_subjects_to_stack(): Return values undocumented

### PR DESCRIPTION
**Pull request by shahsb**
…

In the man page for SSL_add_dir_cert_subjects_to_stack(), the functions returning int have undocumented return values.

https://www.openssl.org/docs/man3.2/man3/SSL_add_dir_cert_subjects_to_stack.html

#23171

##### Checklist

- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: doc/man3/SSL_load_client_CA_file.pod [hunk 1]
```diff
@@ -66,6 +66,14 @@ The operation failed, check out the error stack for the reason.
...
 Pointer to the subject names of the successfully read certificates.
 
+=item 0
+
+The operation failed.
+
+=item 1
+
+The operation succeeded.
+
```
Comment (t8m) [inline doc/man3/SSL_load_client_CA_file.pod:76]: I think the list should be split in two and functions returning the pointer should be listed before the first list and those returning an int should be listed before the second list.

Comment (t8m) [reply doc/man3/SSL_load_client_CA_file.pod:76]: Unfortunately I do not think it will be acceptable with CLA: trivial though.

Comment (shahsb) [reply doc/man3/SSL_load_client_CA_file.pod:83]: > I think the list should be split in two and functions returning the pointer should be listed before the first list and those returning an int should be listed before the second list.

Done. I've addressed the review comment in the latest commit.

File: doc/man3/SSL_load_client_CA_file.pod [hunk 2]
```diff
@@ -66,6 +67,20 @@ The operation failed, check out the error stack for the reason.
 
 Pointer to the subject names of the successfully read certificates.
 
+
```
Comment (t8m) [inline doc/man3/SSL_load_client_CA_file.pod:70]: You'll need to add a line with `=back` above this

**Review by t8m — CHANGES_REQUESTED** [review]

File: doc/man3/SSL_load_client_CA_file.pod (hunk 2 above)
Comment (shahsb) [reply doc/man3/SSL_load_client_CA_file.pod:71]: Done. Thanks!

**Review by t8m — APPROVED** [review]

**Comment by shahsb** [comment]
@t8m - Build fuzzers is failing from long time. Request your guidance for the same.

**Comment by t8m** [comment]
> @t8m - Build fuzzers is failing from long time. Request your guidance for the same.

Please just ignore it for now.

**Review by beldmit — APPROVED** [review]
LGTM

**Comment by t8m** [comment]
Squashed and merged to the master, 3.2, 3.1 and 3.0 branches. Thank you for your contribution.

**Pull request closed without merging**